reporter-web-app/src/utils/index.js

```diff
@@ -40,4 +40,54 @@ const randomStringGenerator = (length = (Math.floor(Math.random() * 501) + 20))
     return alpha(l, symbol(1, alpha(r, []))).join('');
 };
 
-export { randomStringGenerator };
+// Utility function to generate color for given license string 
+const licenseToHslColor = (license) => {
+    let hash = 0;
+    let min;
+    let max;
+    // Basic license classification based on https://scancode-licensedb.aboutcode.org
```

Comment:
IMO we shouldn't do this. I understand that we cannot easily use our own `license-classifications.yml` as its freely defined categories have no machine-readable semantics attached. But we should stick to our principle to not give any advice and not do any interpretation, even if it's just coloring according to something "dangerours" or "safe".

*Edit:* But maybe we could add a `color` property to the categories in `license-classifications.yml`, and make use of them here.

Reply:
Agreed, removed the code in question and created https://github.com/oss-review-toolkit/ort/issues/4150